### PR TITLE
Plans 2023 : Adds a dummy downgrade button with a tooltip for eligible plans

### PR DESCRIPTION
**Pull request by ddc22**
#### Proposed Changes

* This is a temporary change for the initial release, in the next release we will be adding a link to the pre-filled contact support form.

image

#### Testing Instructions
- Purchase a personal plan or higher
- Go to /start/plans?flags=onboarding/2023-pricing-grid
- Make sure the above downgrade buttons are visible

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Fixes Automattic/martech#1459

### CODE REVIEW

**Review by claudiucelfilip — APPROVED** [review]
Works well.

LGTM

File: client/my-sites/plan-features-2023-grid/actions.tsx [hunk 1]
```diff
@@ -168,12 +183,23 @@ const LoggedInPlansFeatureActionButton = ( {
...
-		return (
-			<Button className={ classes } disabled={ true }>
-				{ buttonText }
-			</Button>
-		);
+	const is2023OnboardingPricingGrid = isEnabled( 'onboarding/2023-pricing-grid' );
+	if ( ! availableForPurchase ) {
+		if ( is2023OnboardingPricingGrid ) {
+			return (
+				<Plans2023Tooltip text={ translate( 'Please, contact support to downgrade your plan' ) }>
```
Comment (aneeshd16) [inline client/my-sites/plan-features-2023-grid/actions.tsx]:
```suggestion
				<Plans2023Tooltip text={ translate( 'Please contact support to downgrade your plan.' ) }>
```

Comment (aneeshd16) [reply client/my-sites/plan-features-2023-grid/actions.tsx]: Minor text suggestion ^

**Review by aneeshd16 — APPROVED** [review]
LGTM, with minor copy suggestion.

File: client/my-sites/plan-features-2023-grid/actions.tsx (hunk 1 above)
Comment (ddc22) [reply client/my-sites/plan-features-2023-grid/actions.tsx]: @lucasmendes-design I am going with this suggestion 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7787685

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7787685).

**Pull request merged**